src/OSSupport/Errors.cpp

```diff
@@ -5,13 +5,13 @@
 
 AString GetOSErrorString( int a_ErrNo)
 {
-	char buffer[ 1024 ];
 	AString Out;
 
 	#ifdef _WIN32
 
+	wchar_t buffer[1024];
 	FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM, nullptr, a_ErrNo, 0, buffer, ARRAYCOUNT(buffer), nullptr);
-	Printf(Out, "%d: %s", a_ErrNo, buffer);
+	Printf(Out, "%d: %S", a_ErrNo, buffer);
```

Comment:
It is possible to fix the console with more custom win32 code, as utf8 is a valid codepage (65001). We could set the codepage to that on startup, and then restore on shutdown. But yes, this is a big change.